ui/vite.config.ts

```diff
@@ -1,7 +1,17 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
-
-// https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  resolve: {
```

Comment:
this is so the uI can resolve the noble hashes deps that the workspace defines

Reply:
I don't understand. I can't find any occurrences of `@noble/hashes` in the ui directory.

Reply:
its a sub-dependency from `@cosmjs/proto-signing` 🫠 

```console
✘ [ERROR] Could not resolve "@noble/hashes/sha3"

    ../node_modules/@cosmjs/proto-signing/node_modules/@cosmjs/crypto/build/keccak.js:4:23:
      4 │ const sha3_1 = require("@noble/hashes/sha3");
        ╵                        ~~~~~~~~~~~~~~~~~~~~

  The module "./sha3.js" was not found on the file system:

    ../node_modules/@noble/hashes/package.json:116:17:
      116 │       "require": "./sha3.js"
          ╵                  ~~~~~~~~~~~
```

First encountered [here](https://github.com/Agoric/dapp-orchestration-basics/pull/18#issuecomment-2293634136)